frontend/src/i18n/locales/en.yaml

```diff
@@ -385,6 +385,20 @@ manage:
       referencing-pages: Referencing pages
       referencing-pages-explanation: 'This video is referenced on the following pages:'
       no-referencing-pages: No pages reference this video.
+      delete:
+        title: Delete Video
+        confirm: Delete Video?
+        cannot-be-undone: This action <strong>cannot</strong> be undone!
+        failed: Deleting the video failed.
```

Comment:
I think this `failed` key is unused and can be removed

Reply:
This is shown in the modal if the mutation returns an error (lines 107-108 in `Details.tsx`).

Reply:
Ah yes